kola/tests/util/devcontainer.go

```diff
@@ -0,0 +1,185 @@
+// Copyright The Mantle Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package util
+
+import (
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+	"unicode"
+
+	"github.com/flatcar/mantle/kola"
+	"github.com/flatcar/mantle/kola/cluster"
+	"github.com/flatcar/mantle/platform"
+	"github.com/flatcar/mantle/platform/conf"
+	"github.com/flatcar/mantle/platform/local"
+	"github.com/flatcar/mantle/platform/machine/qemu"
+	"github.com/flatcar/mantle/platform/machine/unprivqemu"
+	mutil "github.com/flatcar/mantle/util"
+)
+
+type (
+	downloadLibraryParameters struct {
+		ImageDirectoryURLTemplate string
+	}
+)
+
+var (
+	downloadLibraryTemplate = TrimLeftSpace(`
```

Comment:
While having this is better than having to remember to add all the required `curl` flags, I think the most straightforward way in mantle is to let Ignition download things for you.
So, this function could emit the URL for use in the Ignition configuration. The unpacking with lbzip2 would have to be done on the host still but won't really need to be a template/function.